### ISSUE TITLE
read is not the inverse of write for MESS keyword

### ISSUE THREAD

**Issue by eivindjahren** [equinor/ecl-data-io#19]
When reading a written `MESS` keyword, the result is a tuple containing empty data. This means that re-writing that data results in a different file:
```python
import ecl_data_io as eclio

eclio.write("file.ecl", [("MESSHEAD", eclio.MESS)])
data = eclio.read("file.ecl") # [("MESSHEAD", np.array([], dtype=np.float64))]

eclio.write("file.ecl", data) # results in a file with "MESSHEAD" having type DOUB
                              # and not type MESS

**Comment by mariuswinger** [comment]
Closed by #20?

**Comment by eivindjahren** [comment]
Indeed closed by #20
